content/commands/cluster-migration.md

```diff
@@ -0,0 +1,208 @@
+
+```bash
+CLUSTER MIGRATION STATUS ID 24cf41718b20f7f05901743dffc40bc9b15db339
+```
+
+Cancel a specific migration task:
+
+```bash
+CLUSTER MIGRATION CANCEL ID 24cf41718b20f7f05901743dffc40bc9b15db339
+```
+
+Cancel all migration tasks:
+
+```bash
+CLUSTER MIGRATION CANCEL ALL
+```
+
+## Return information
+
+{{< multitabs id="return-info"
+    tab1="RESP2"
+    tab2="RESP3" >}}
+
+For the `IMPORT` subcommand:
+[Bulk string reply](../../develop/reference/protocol-spec#bulk-strings): task ID on success, or error message on failure.
+
+For the `CANCEL` subcommand:
+[Integer reply](../../develop/reference/protocol-spec#integers): number of cancelled tasks.
+
+For the `STATUS` subcommand:
+[Array reply](../../develop/reference/protocol-spec#arrays): a list of migration task details. Each task is represented as an array containing field-value pairs:
+- `id`: Task identifier
+- `slots`: Slot range being migrated
+- `source`: Source node ID
+- `dest`: Destination node ID
+- `operation`: Operation type (typically "migrate")
+- `state`: Current state ("completed", "in_progress", etc.)
+- `last_error`: Last error message (empty if none)
+- `retries`: Number of retry attempts
+- `create_time`: Task creation timestamp
+- `start_time`: Task start timestamp
+- `end_time`: Task completion timestamp (if completed)
+- `write_pause_ms`: Write pause duration in milliseconds
+
+-tab-sep-
+
+For the `IMPORT` subcommand:
+[Bulk string reply](../../develop/reference/protocol-spec#bulk-strings): task ID on success, or error message on failure.
+
+For the `CANCEL` subcommand:
+[Integer reply](../../develop/reference/protocol-spec#integers): number of cancelled tasks.
+
+For the `STATUS` subcommand:
+[Array reply](../../develop/reference/protocol-spec#arrays): a list of migration task details. Each task is represented as an array containing field-value pairs:
+- `id`: Task identifier
+- `slots`: Slot range being migrated
+- `source`: Source node ID
+- `dest`: Destination node ID
+- `operation`: Operation type (typically "migrate")
+- `state`: Current state ("completed", "in_progress", etc.)
+- `last_error`: Last error message (empty if none)
+- `retries`: Number of retry attempts
+- `create_time`: Task creation timestamp
+- `start_time`: Task start timestamp
+- `end_time`: Task completion timestamp (if completed)
+- `write_pause_ms`: Write pause duration in milliseconds
+
+{{< /multitabs >}}
+
+## Notes
+
+- The atomic slot migration feature is available starting from Redis 8.4.0.
+- Migration tasks are tracked in memory and the number of archived tasks is controlled by the `cluster-slot-migration-max-archived-tasks` configuration (default: 32).
+- During migration, the source node may pause writes briefly to hand off slot ownership. The duration is controlled by `cluster-slot-migration-write-pause-timeout` (default: 10 seconds).
+- The handoff threshold is controlled by `cluster-slot-migration-handoff-max-lag-bytes` (default: 1MB).
```

Comment:
maybe we should remove all the configs from "Notes" section. There is a config section down below with relatively better explanations.